src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+    public  static void main(String[] args) {
```

Comment:
The access modifier 'public' should be omitted for the 'main' method since it's unnecessary in this context. The 'main' method can be package-private as it's not intended to be used outside of its package.